src/biotite/structure/info/radii.py

```diff
@@ -26,37 +26,106 @@
     ("S",  1, 0) : 1.77,
     ("S",  2, 0) : 1.77, # Not official, added for completeness (MET)
     ("S",  2, 1) : 1.77,
-    ("F",  1, 0) : 1.47, # Taken from _SINGLE_RADII
-    ("CL", 1, 0) : 1.75, # Taken from _SINGLE_RADII
-    ("BR", 1, 0) : 1.85, # Taken from _SINGLE_RADII
+    ("F",  1, 0) : 1.47, # Taken from _SINGLE_ELEMENT_VDW_RADII
+    ("CL", 1, 0) : 1.75, # Taken from _SINGLE_ELEMENT_VDW_RADII
+    ("BR", 1, 0) : 1.85, # Taken from _SINGLE_ELEMENT_VDW_RADII
     ("I",  1, 0) : 1.98, # Taken from _SINGLE_RADII
 }
 
-_SINGLE_RADII = {
-    "H":  1.20,
+_SINGLE_ELEMENT_VDW_RADII = {
```

Comment:
Just an optional nit: I think *single atom* is more appropriate than *single element* here.

```suggestion
_SINGLE_ATOM_VDW_RADII = {
```

Reply:
Done ✅